### ISSUE TITLE
dnLogNormal help inaccurate 

### ISSUE THREAD

**Issue by ssolo** [revbayes/revbayes#137]
mu <- 1.
sigma <- 1.
for (j in 1:10000) {x[j]~dnLnorm(mu,sigma)}
mean(x)
   4.507144
var(x)
   36.43327

The mean is exp(mu+sigma^2/2) and the variance is (exp(sigma^2)-1)*exp(2*mu+sigma^2).

The help on the other hand states the following:
  
Arguments

       mean : The mean in log-space (observed mean is exp(m)).
              Type:       Real, <any>, const reference
         sd : The standard deviation in log-space.
              Type:       RealPos, <any>, const reference

**Comment by ferroao** [comment]
I think the help is accurate, try:
for (j in 1:10000) {thelog[j]=ln(x[j])}
mean(thelog)
stdev(thelog)

**Comment by ssolo** [comment]
Yes mean( ln(X) ) = mu, but the help states that "observed mean is exp(m)”, where as the observed mean is in fact exp(mu+sigma^2/2), this certainly seems inaccurate. I would also call “mean" “mu" and “sd" “sigma", as per the usual parametrisation.   

cf.

Arguments
       mean : The mean in log-space (observed mean is exp(m)).
              Type:       Real, <any>, const reference
         sd : The standard deviation in log-space.
              Type:       RealPos, <any>, const reference

> mu <- 1.
> sigma <- 1.
> for (j in 1:10000)
+ {
+ x[j]~dnLnorm(mu,sigma)
+ lnx[j] = ln(x[j])
+ }
> mean(x)
   4.465145
> mean(lnx)
   0.9967692
> mu <- 1.
> sigma <- 1.
> for (j in 1:10000)
+ {
+ x[j]~dnLnorm(mu,sigma)
+ lnx[j] = ln(x[j])
+ }

> exp(mu)
   2.718282

> exp(mu+sigma^2/2)
   4.481689

> mean(x)
   4.400285

> mean(lnx)
   0.9854552

> sqrt((exp(sigma^2)-1)*exp(2*mu+sigma^2))
   5.874744

> stdev(x)
   6.030914

> stdev(lnx)
   0.9893643
